src/client/app/comm.ts

```diff
@@ -1,17 +1,25 @@
+import { BaseModel } from '../model/BaseModel';
+import { Content } from '../model/Content';
 import { Problem } from '../model/Problem';
 import { ProblemResponse } from '../model/ProblemResponse';
 
+export const CONTENT_CLASS: Content = new Content().constructor as any;
+export const PROBLEM_CLASS: Problem = new Problem().constructor as any;
```

Comment:
Could you please explain me this part?
Isn't the `getInstance` method static? Why do you need a `new` ?
Or maybe I missed something.

Reply:
FYI, it's not the final push, this one was made to share the progress with @adjamenhmh ;)
I'm going to refactor that part. I just needed a quick implementation. i'm going to push accessors into the BaseModel class, so these hacks won't be visible :D

Reply:
Fixed: 
-`getInstance()` used now in place of `new` to get new entity instances
- CONTENT_CLASS (resp. PROBLEM_CLASS) is defined in the Content.ts (resp. Problem.ts) file.